_config.yml

```diff
@@ -11,7 +11,12 @@ markdown: kramdown
 
 # Custom Variables
 
+main:
+    plugins: "assets/plugins"
+    css: "assets/css"
+    js: "assets/js"
+
 path:
```

Comment:
Also I think you can access it with `{{ page.permalink }}` anything you set in the front matter at the top is a configuration or variable just like in the _config.yml. Anything in config.yml is `site.*` and anything in the page is `page.*`